content/reference/sql/version-control/dolt-sql-procedures.md

```diff
@@ -487,6 +488,60 @@ CALL DOLT_ADD('table')
 CALL DOLT_RESET('table')
 ```
 
+
+## `DOLT_REMOTE()`
+
+Adds a remote for a repository at given url, or removes an existing remote with its remote-tracking branches 
+and configuration settings. Works exactly like [`dolt remote`](../../cli.md#dolt-remote) on the CLI, and takes 
+the same arguments except for listing tags. To list existing tags, use 
+[`dolt_remotes`](./dolt-system-tables.md#dolt_remotes) system table.
```

Comment:
(minor) you might consider including "system table" in the link text, just because the backtick formatting makes it harder to see links.